<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ride Comparison Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="routeLengthInput">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter route length (m)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numberOfCarsInput">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter number of cars</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="manualSpacingRadioButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Manual Traffic Light Spacing</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="randomSpacingRadioButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Random Traffic Light Spacing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startSimulationButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Simulation</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="simulationView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>800</width>
      <height>410</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
